### PR TITLE
Modify S5747: LaYC format

### PR DESCRIPTION
**Pull request by guillaume-dequenne-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S5747/python/rule.adoc [hunk 1]
```diff
@@ -1,60 +1,67 @@
+This rule raises an issue when a bare raise` statement is not in an `except` or `finally` block.
```
Comment (jean-jimbo-sonarsource) [inline rules/S5747/python/rule.adoc:1]: Missing backtick for raise: This rule raises an issue when a bare `raise` statement

**Review by jean-jimbo-sonarsource — CHANGES_REQUESTED** [review]
small typo, otherwise OK

File: rules/S5747/python/rule.adoc [hunk 2]
```diff
@@ -1,60 +1,67 @@
...
-        raise
-    else:
-        raise ValueError()
-    finally:
+    except ValueError:
         raise
 ----
 
-
 == Resources
```
Comment (joke1196) [inline rules/S5747/python/rule.adoc:66]: Documentation section is missing

**Review by joke1196 — CHANGES_REQUESTED** [review]
Looks good! I like the note section it makes a lot of sense. I have just added a small comment.

**Review by joke1196 — APPROVED** [review]
LGTM!

**Review by jean-jimbo-sonarsource — APPROVED** [review]

**Pull request merged**